divHretention/plotting/plot_along_divertor.py

```diff
@@ -250,3 +250,21 @@ def __init__(self, **kwargs):
         super().__init__(
             quantities=quantities,
             **kwargs)
+
+
+def plot_inv_with_uncertainty(x, y, sigma, alpha_fill=0.3, **kwargs):
+    """Plots the inventory along the divertor with the associated uncertainity
+    filled
+
+    Args:
+        x ([type]): Arc length (m) along the divertor
+        y ([type]): Inventory per unit thickness (H/m) 
+        sigma ([type]): Relative error
```

Comment:
You need to specify the type here (and not leave `[type]`).
```suggestion
        x (numpy.array): Arc length (m) along the divertor
        y (numpy.array): Inventory per unit thickness (H/m) 
        sigma (numpy.array): standard deviation
```